gdal/frmts/pds/pds4dataset.cpp

```diff
@@ -4106,6 +4106,8 @@ PDS4Dataset *PDS4Dataset::CreateInternal(const char *pszFilename,
             else if( EQUAL(pszDriverName, "FITS") )
             {
                 osHeaderParsingStandard = "FITS 3.0";
+                papszOptions = CSLSetNameValue(
+                    papszOptions, "VAR_VERTICAL_DISPLAY_DIRECTION", "Bottom to Top");
```

Comment:
After much discussion with @cmarmo offline (mostly me getting on the same page for FITS), I think this is fine for a temporary update. However, I am worried about the need for all templates to have the template variable VERTICAL_DISPLAY_DIRECTION. This is not a guarantee and I think could easily lead to bad PDS4 labels. 

A more robust solution I think would need GDAL to build the DISPLAY section in the PDS4 label depending on the input format.  This seems to only be a problem for GDAL's new PDS4 option `CREATE_LABEL_ONLY`

FITS should always be "Bottom to Top". Whereas the current default, "Top to Bottom" can remain for other `CREATE_LABEL_ONLY` formats (GeoTIFF, ISIS3, VICAR, PDS3 and ENVI). Even though that is the default, I can think of some potential display direction for those formats -- even GeoTIFF). That is for another time. If the ability to to use `CREATE_LABEL_ONLY` on a format becomes too complicated, then we could also turn that capability off. This just means one would need to convert the input format to new file for PDS4. I will happily write up this need up in a new issue to be tackled at a later time.

Reply:
> However, I am worried about the need for all templates to have the template variable VERTICAL_DISPLAY_DIRECTION. This is not a guaranteed variable within all templates and I think could easily lead to bad PDS4 labels.

I've slightly altered @cmarmo work (in #3520) to not require the template variable VERTICAL_DISPLAY_DIRECTION but to edit directly the template by finding the corresponding XML node and changing its text value. This should work hopefully with all valid PDS4 templates

Reply:
and if the XML node is missing a "Warning 1: PDS4 template lacks a disp:vertical_display_direction element where to write Bottom to Top" is emitted